File: app/src/main/java/com/wq/TestActivity.java
package com.wq;

import android.Manifest;
import android.os.Bundle;

import com.example.utillibrary.PermissionUtil;
import com.wq.allandroid.R;

import androidx.appcompat.app.AppCompatActivity;

public class TestActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);
        PermissionUtil.INSTANCE.checkPermission(this, new String[]{
                Manifest.permission.READ_EXTERNAL_STORAGE
        }, 0, new PermissionUtil.OnPermissonCallback() {
            @Override
            public void isGrant(boolean grant) {

            }
        });
    }
}

File: app/src/main/java/com/wq/TestActivity.kt
package com.wq

import android.icu.text.TimeZoneNames
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.wq.allandroid.R
import java.util.*

class TestActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)
        val decorView = window.decorView
        TimeZone.getAvailableIDs().forEach {
            TimeZone.getTimeZone(it)
        }
        TimeZone.getTimeZone("8").displayName
//        TimeZoneNames.getInstance(Locale.CHINA).getAvailableMetaZoneIDs()
        val option = View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN or View.SYSTEM_UI_FLAG_LAYOUT_STABLE
    }
}